codice/ddf
# DDF-4081 Addressed ErrorProne JUnit4TestNotRun errors in catalog-core-standardframework

#### What does this PR do?
This PR
- Addresses ErrorProne JUnit4TestNotRun errors in catalog-core-standardframework
- Updates the maven-surefire-plugin configuration in catalog-core-standardframework so that Spock tests are not skipped and fixes two broken Spock tests
#### Who is reviewing it? 
@peterhuffer @kcover 
#### Select relevant component teams: 
@codice/test 
#### Ask 2 committers to review/merge the PR and tag them here.
@coyotesqrl
@clockard 
#### How should this be tested?
- CI build
- Confirm that Spock tests are being run in `catalog-core-standardframework` during c`mvn clean install`. You can modify one of the Spock tests to fail and confirm that the `mvn clean install` fails.
#### What are the relevant tickets?
[DDF-4081](https://codice.atlassian.net/browse/DDF-4081)

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
